flatpak/README.md

```diff
@@ -0,0 +1,37 @@
+# Eclipse Flatpak
+
+## Installation
+
+To install the Eclipse Flatpak via the command line:
+
+* Add the Eclipse Nightly repo
+
+    `flatpak remote-add --if-not-exists eclipse-nightly https://download.eclipse.org/linuxtools/flatpak-I-builds/eclipse.flatpakrepo`
+
+- To check the available products and branches run:
+
+    `flatpak remote-ls eclipse-nightly`
+  
+- Install your chosen product by running:
+
+    `flatpak install eclipse-nightly org.eclipse.Platform` or
+    `flatpak install eclipse-nightly org.eclipse.Sdk`
+
+    This will prompt you to chose a branch or abort, to skip this you can specify a branch by adding `/x86_64/[branch]` to the command:
```

Comment:
Actually you can omit the architecture, e.g.:
```
flatpak install eclipse-nightly org.eclipse.Sdk//master
```
And by default it will use the correct architecture for your machine.